.gitignore

```diff
@@ -1,3 +1,7 @@
+# locale
+example/.*
```

Comment:
It was just to remove the hidden folders like the one created by jupyter notebook like .ipynb_checkpoints for example.